src/components/back-to-top/back-to-top.component.jsx

```diff
@@ -34,11 +34,9 @@ class BackToTop extends React.Component {
 
   render() {
     return (
-      <div>
-        <button className="back-to-top" onClick={this.scrollingMagic}>&uarr;</button>
-      </div>
+      <button className="back-to-top" onClick={this.scrollingMagic}>&uarr;</button>
```

Comment:
no need for `<div>`. keep the node depth at 1

Reply:
I thought that a <div> had to be the highest element in a component. Is that not the case?